components/src/core/components/ProfilePic/ProfilePic.vue

```diff
@@ -55,6 +60,23 @@ export default defineComponent({
         return TARGETS.indexOf(value) !== -1;
       },
     },
+    header: {
```

Comment:
This is not correct. ProfilePic component is not a part of the cardtable. Cardtable is just using it as component for its cells. hence headers and rowitems should not be passed here

Reply:
If you must use a generic event to communicate the click to cell